tests/unit/overlays/__init__.py

```diff
@@ -0,0 +1,26 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from craft_parts.features import Features
+
+
+def setup_module():
```

Comment:
I think when we tackle CRAFT-1604 we should also remove this setup/teardown duplication across many test files

Reply:
My plan is to have tests that require the overlay feature to be enabled under `tests/unit/features/overlay` and then enable it globally for the entire package. The only exception would be the `tests/unit/overlay` because it has a corresponding package under `craft_parts`.